public/scripts/glitch.js

```diff
@@ -0,0 +1,77 @@
+const headerLogoEl = document.getElementById("header-logo");
+
+headerLogoEl.addEventListener(
+  "click",
+  event => {
+    event.preventDefault();
+    setUpGlitchCounter(headerLogoEl);
+  },
+  { once: true }
+);
+
+function setUpGlitchCounter(logoElement) {
+  createCounterTemplate(logoElement);
+  const counterEl = document.getElementById("glitch-counter");
+
+  const secretDate = new Date(2019, 9, 1);
```

Comment:
Pls change it to 20.09 and the we can merge it

Reply:
Or you can just replace it with milliseconds. Then it will show number with minus, when we skip the deadline :)